sql/parser/parse_test.go

```diff
@@ -614,6 +614,13 @@ CREATE TABLE a (b INT DEFAULT (SELECT 1))
                                         ^
 `,
 		},
+		{
+			`SELECT POSITION('high', 'a')`,
```

Comment:
Another option could be to parse `POSITION('ig' in 'high')` directly to `STRPOS('high', 'ig')`, because POSITION is basically just a form of the more standard looking STRPOS function. I'm not sure if this would be considered a hack or a nice workaround.